src/comestible_inv.cpp

```diff
@@ -0,0 +1,780 @@
+        return;
+    }
+
+    char sym = minimap_get_sym();
+    if( sym != '\0' ) {
+        auto sq = pane->get_area();
+        auto pt = pc + sq->offset;
+        // invert the color if pointing to the player's position
+        auto cl = sq->info.type == comestible_inv_area_info::AREA_TYPE_PLAYER ?
+                  invert_color( c_light_cyan ) : c_light_cyan.blink();
+        mvwputch( minimap, pt.xy(), cl, sym );
+    }
+
+    // Invert player's tile color if exactly one pane points to player's tile
+    bool player_selected = pane->get_area()->info.type == comestible_inv_area_info::AREA_TYPE_PLAYER;
+    g->u.draw( minimap, g->u.pos(), player_selected );
+    wrefresh( minimap );
+}
+
+char comestible_inventory::minimap_get_sym() const
+{
+    if( pane->get_area()->info.type == comestible_inv_area_info::AREA_TYPE_PLAYER ) {
+        return '^';
+    } else if( pane->is_in_vehicle() ) {
+        return 'V';
+    } else {
+        return ' ';
+    }
+}
+
+void comestible_inventory::do_return_entry()
+{
+    // only save pane settings
+    g->u.assign_activity( activity_id( "ACT_COMESTIBLE_INVENTORY" ) );
+    g->u.activity.auto_resume = true;
+    uistate.comestible_save.exit_code = exit_re_entry;
+    save_settings();
+}
+
+void comestible_inventory::refresh( bool needs_recalc, bool needs_redraw )
+{
+    recalc = needs_recalc;
+    pane->needs_recalc = needs_recalc;
+
+    redraw = needs_redraw;
+    pane->needs_redraw = needs_redraw;
+}
+
+void comestible_inventory_food::init()
+{
+    pane = new comestible_inventory_pane_food();
+    comestible_inventory::init();
+}
+
+input_context comestible_inventory_food::register_actions()
+{
+    input_context ctxt = comestible_inventory::register_actions();
+    ctxt.register_action( "SWITCH_FOOD" );
+    ctxt.register_action( "HEAT_UP" );
+    ctxt.register_action( "CRAFT_WITH" );
+    ctxt.register_action( "SHOW_HELP" );
+    return ctxt;
+}
+
+void comestible_inventory_food::set_additional_info( std::vector<legend_data> data )
+{
+    std::pair<std::string, nc_color> desc;
+    desc = g->u.get_hunger_description();
+    data.emplace_back( string_format( "%s %s", _( "Food :" ), colorize( desc.first, desc.second ) ),
+                       point_zero, c_dark_gray );
+    desc = g->u.get_thirst_description();
+    data.emplace_back( string_format( "%s %s", _( "Drink:" ), colorize( desc.first, desc.second ) ),
+                       point_south, c_dark_gray );
+    desc = g->u.get_pain_description();
+    data.emplace_back( string_format( "%s %s", _( "Pain :" ), colorize( desc.first, desc.second ) ),
+                       point( 0, 2 ), c_dark_gray );
+    comestible_inventory::set_additional_info( data );
+}
+
+std::string comestible_inventory_food::process_actions( input_context ctxt )
+{
+    const std::string action = comestible_inventory::process_actions( ctxt );
+
+    //using ai = comestible_inv_area_info;
```

Comment:
Please don't commit commented out code. Either it's useful code and should be run, so it should not be a comment at all. Or it is useless code that should not be there in the first place.